lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -73,7 +73,7 @@ class RenderBlockingResources extends Audit {
       // This audit also looks at CSSUsage but has a graceful fallback if it failed, so do not mark
```

Comment:
do you want to change anything about `CSSUsage` for this audit, since after this PR it won't ever see it unless we get around to optionalArtifacts?

Reply:
oh yeah nice catch!